pandas/io/common.py

```diff
@@ -47,10 +46,10 @@
     from contextlib import contextmanager, closing  # noqa
     from functools import wraps  # noqa
 
-    # @wraps(_urlopen)
+    # @wraps(urlopen2)
     @contextmanager
     def urlopen(*args, **kwargs):
-        with closing(_urlopen(*args, **kwargs)) as f:
+        with closing(urlopen2(*args, **kwargs)) as f:
```

Comment:
I'm not sure this workaround is used anywhere.
Maybe we can remove this?

Reply:
NP. I'll leave it there. Just renamed the `urlopen` variants to try to make it a little bit clear of what is going on.